crypto/blake256/hasher256.go

```
@@ -0,0 +1,280 @@
+// Callers may optionally choose to call [WriteBytes] which does not return an
+// error to make the fact writing can never fail.
+func (h *Hasher256) Write(b []byte) (int, error) {
+	return h.h.write(b)
+}
+
+// WriteBytes adds the given bytes to the rolling hash.
+//
+// This method is identical to [Write] except it does not return an error in
+// order to make it clear that writing can never fail.
+func (h *Hasher256) WriteBytes(b []byte) {
+	h.h.write(b)
```

golangci-lint (2.12.2)

[error] Error return value of `h.h.write` is not checked

(errcheck)